demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1918,6 +2014,14 @@ export class DemoMeetingApp
     let content = '';
     let attendee: Attendee = null;
     for (const item of result.alternatives[0].items) {
+      let transcriptWord = item.content;
+      if (item.hasOwnProperty('confidence') && !item.content.startsWith("[") && item.confidence < 0.5) {
+        item.content = `<span class='confidence-style'>${item.content}</span>`;
```

Comment:
Ahem:

```
09:32:26.986 const item = { content: 'Jack & Jill' };
09:32:27.006 undefined
09:32:37.134 item.content = `<span class='confidence-style'>${item.content}</span>`;
09:32:37.148 "<span class='confidence-style'>Jack & Jill</span>"
```

You have some HTML escaping concerns here.

Rather than replacing strings in place, changing their 'type' from a string to HTML, you should separate these phases and use DOM operations.